source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -344,7 +386,6 @@ Before a connection can be returned outside of the pool, it must be “set up”
       perform connection handshake
       handle OP_COMPRESSED
       perform connection authentication
-      mark connection as readyToUse
```

Comment:
a connection is now marked as "available" or "in use" depending on the context it's created in. No need for an additional state.